docs/spark-version-update-process.md

```diff
@@ -0,0 +1,54 @@
+# Updating the base image Spark version
+
+This document describes the general workflow for updating the Apache Spark
+version present in the base image. This guide follows the process for
+installing the default binary archives as distributed by the
+[Spark project](https://spark.apache.org).
+
+## prerequisites
+
+* shell access
+* an editor available
+* access to the `docker` command line tool and a registry (for testing)
+* [cekit](https://cekit.readthedocs.io/en/latest/) available
+
+## procedure
+
+1. update version and download link in `image.yaml`
```

Comment:
reference the change-yaml.sh utility script here?

Reply:
yes, this was just an oversight on my part. i will go back and add the script references.